acceptance_tests/features/steps/display_respondent_alongside_response_status.py

```diff
@@ -0,0 +1,122 @@
+import logging
+
+import time
+from behave import given, when, then
+from structlog import wrap_logger
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import reporting_unit, collection_exercise_details, \
+    change_response_status, surveys_todo
+from acceptance_tests.features.pages.surveys_history import go_to_history_tab
+from acceptance_tests.features.steps.authentication import signed_in_internal, signed_in_respondent
+from acceptance_tests.features.steps.change_response_status import go_to_reporting_unit_page
+from controllers.collection_exercise_controller import map_ce_status
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@given('a user has completed a survey online')
+def enrolling_onto_survey(context):
+    signed_in_respondent(context)
+    browser.click_link_by_id('access_survey_button_1')
+    collection_exercise_details.upload_collection_instrument(
+        test_file='resources/collection_instrument_files/064_201803_0001.xlsx')
+    browser.find_by_id('upload_survey_button').click()
+
+
+@given('an external user is enrolled onto a given survey')
+def enrolled_respondent_but_not_completed(context):
+    signed_in_respondent(context)
+
+
+@given('a user has completed a survey via the telephone')
+def user_completes_survey_by_telephone(context):
+    signed_in_internal(())
+    go_to_reporting_unit_page(context)
+    reporting_unit.click_data_panel(context.short_name)
+    reporting_unit.click_change_response_status_link(survey=context.short_name, period=context.period)
+    mapped_status = map_ce_status('Completed by phone')
+    change_response_status.update_response_status(mapped_status)
+
+
+@given('the status is set to "Completed by phone"')
+def survey_is_set_to_completed_by_telephone(context):
+    signed_in_respondent(context)
+    go_to_history_tab()
+    assert browser.find_by_text('Completed by phone')
+
+
+@given('the status is set to "No longer required"')
+def survey_set_to_longer_required(context):
```

Comment:
Probably best not to use quotation marks like this . There is a mechanism in behave tests that can pass these into the tests as a variable. So adding quotation marks and NOT passing a variable will lead to confusion down the line . OR you could add the variable and pass the variable to the browser if appropriate.E.g 

either 

@given('the status is set to No longer required') 

or 

@given('the status is set to "No longer required"')
def survey_set_to_longer_required(context, status):
 signed_in_respondent(context)
    go_to_history_tab()
    assert browser.find_by_text(status)